src/main/java/duke/commands/ByeCommand.java

```diff
@@ -0,0 +1,39 @@
+package duke.commands;
+
+import duke.tasks.TaskList;
+import duke.util.Storage;
+import duke.util.Ui;
+
+/**
+ * Represents the command 'bye'.
+ */
+public class ByeCommand extends Command {
+
+    /**
+     * Method to check if command is the exit command.
+     * @return boolean stating if command is an exit command
+     */
+    public boolean isExit() {
+        return true;
+    }
+
+    /**
+     * Method to execute 'bye' command.
+     * @param taskList task list in running program.
+     * @param ui ui handling running program.
+     * @param storage storage handling running program.
+     */
+    public void execute(TaskList taskList, Ui ui, Storage storage) {
+        ui.printExitMsg();
+    }
+
+    /**
+     * Method to compare 2 commands.
+     * @param command object to compare command to.
+     * @return boolean stating if this command and command are equal.
+     */
+    @Override
+    public boolean equals(Object command) {
+        return command instanceof ByeCommand;
+    }
+}
```

Comment:
I might be mistaken but I think the javadoc for the methods should start with e.g. "Executes 'bye' command" instead of a noun.  And just a thought, while the methods are easy to understand so not much explanation is needed, it is a little confusing when you use the same word to imply different objects. Like the command in the equals method. Maybe it would be better if you explained a little more?

Reply:
Also, I think it's quite neat that you have an abstract command class and the other commands inherit from it!

Reply:
Ya for JavaDoc the methods start with a verb.

Reply:
Thank you guys so much for the heads up on the JavaDocs!! Noted with thanks